src/component/ComponentInternalsImpl.ts

```diff
@@ -595,7 +595,7 @@ class ComponentInternalsImpl implements ComponentInternals, Tellable {
 
 	private initFields(): void {
 		this.id = IdGenerator.INSTANCE.generate();
-		this.logger = LoggerFactory.getLogger(`${this.getName()} component ${this.id}`);
+		this.logger = LoggerFactory.getLogger(`Component<${this.getName()}> ${this.id}`);
```

Comment:
Doesn't this make it look like it is a generic type?  I think that the square brackets that you used in ModuleImpl might be less confusing and more consistent.

Reply:
From among the collection of Modules the selected is logging, whereas the Component of type X is saying something, but for consistency, I have changed it.